src/app/lib/config/services/uzbek.ts

```diff
@@ -371,8 +684,324 @@ export const service: UzbekConfig = {
+                heading3: 'Қўшимча, мажбурий бўлмаган дата тўплаш',
+                para6:
+                  'Сиз AMP саҳифаларда дата тўплашга рози ва айни дамда Буюк Британия ташқарисида бўлсангиз биз рекламаларни сизга қанчалар алоқаси бор ёки йўқлигига қараб танлаб, тақдим қиламиз.',
+                para7: {
+                  text: 'BBC ва реклама ҳамкорларимиз рекламаларни шахсийлаштиришга доир кўпроқ маълумот',
+                  url: 'https://www.bbc.com/usingthebbc/cookies/how-does-the-bbc-use-cookies-for-advertising/',
+                },
+                para8:
+                  'Сизга мосланган рекламаларни қабул қилишни истамасангиз, марҳамат қилиб пастдаги "Дата тўплашни рад қилинг ва давом этинг" деган танловни босинг. Унутманг, сиз барибир рекламани кўрасиз, аммо у сиз учун мослаштирилмаган бўлади.',
+                para9:
+                  'Сиз бу шартларни "Реклама танловлари/менинг маълумотларимни сотманг" деган бўлимда исталган вақтда ўзгартиришингиз мумкин.',
+              },
+            },
+          },
+          canonical: {
+            title: 'Кукис(Cookies)га рози эканингизни билдиринг',
+            description: {
+              uk: {
+                first: 'Биз ',
+                linkText: 'кукидан',
+                last: ' сизга янада яхшироқ онлайн имконият бериш мақсадида фойдаланамиз. Марҳамат қилиб ушбу барча кукиларга розилигингизни билдиринг.',
+                linkUrl:
+                  'https://www.bbc.co.uk/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
+              },
+              international: {
+                first: 'Биз ',
+                linkText: 'кукидан',
+                last: ' сизга янада яхшироқ онлайн имконият бериш мақсадида фойдаланамиз. Марҳамат қилиб ушбу барча кукиларга розилигингизни билдиринг.',
+                linkUrl:
+                  'https://www.bbc.com/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
+              },
+            },
+            accept: 'Ҳа, мен розиман',
+            reject: 'Йўқ, мени ўзгартириш саҳифасига етакланг',
+            rejectUrl:
+              'https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
+          },
+        },
+      },
+      media: {
+        noJs: 'Мултимедиа ўйнаш бу қурилмада дастакланмайди',
+        contentExpired: 'Бу контентни ортиқ тинглаб/томоша қилиб бўлмайди.',
+        contentNotYetAvailable:
+          'Бу контент ҳали тинглаш/томоша қилишга тайёр эмас.',
+        audio: 'Аудио',
+        photogallery: 'Фото галерея',
+        video: 'Видео',
+        bbc_uzbek_radio: {
+          title: 'BBC Uzbek Radio Dasturi',
+          subtitle: 'O’zbekiston, mintaqa va dunyo yangiliklari O’zbek tilida',
+        },
+        bbc_uzbek_tv: {
+          title: 'Bi-bi-si O‘zbek - Afg‘oniston uchun TV dastur',
+          subtitle:
+            'Dastur haftada 5 kun - dushanbadan juma kuniga qadar Toshkent vaqti bilan soat 18.30 -18.40 da efirga uzatiladi',
```

Comment:
should be transliterated?

Reply:
Yep all the translations in `lat` are basically the `cyr` version, so it will be up to editorial to update these.

We aren't in a _worse_ position that we currently are with this change. This change will just make it easier for editorial to update the translations as the spread syntax probably won't mean much to them.

Reply:
Although this looks like the wrong language to begin with?